data/categories.txt

```diff
@@ -16694,3 +16694,8 @@ uk:Центр мистецтв
 vi:Trung tâm nghệ thuật
 zh-Hans:艺术中心
 zh-Hant:藝術中心
+
+shop-craft
```

Comment:
```suggestion
shop-craft|@shop
```
It provides "shop"/"store" etc. synonyms automatically, so all lang-specific lines shouldn't contain them (pls check how synonyms are done for other shops).

Please take existing `[type.shop.craft]` translations from `types_strings.txt` as a starting point for all languages.

Reply:
Please take existing [type.shop.craft] translations from types_strings.txt as a starting point for all languages.

Reply:
@ProgramminCat just checking to make sure - have you seen this comment?

Reply:
> @ProgramminCat just checking to make sure - have you seen this comment?

Yes I have. I will try that soon